### ISSUE TITLE
[regression] MFVec3f fields with trailing comma not working after 4.2.4

### ISSUE THREAD

**Issue by splace** [create3000/x_ite#31]
[example.zip](https://github.com/create3000/x_ite/files/2473674/example.zip)

works in Titania

**Comment by create3000** [comment]
[example.zip](https://github.com/create3000/x_ite/files/2474758/example.zip)
Works for me, tested in Firefox.

**Comment by splace** [comment]
Sorry, odd coincidence and me being slow, it turns out to be nothing to do with protos/files etc.

its simply when you have an extra 'redundant' comma at the end of a an MFVec3f, in this case for an IFS's coordinates.

(in the example the Box was a late change from an IFS, to cut down the size, i assumed wouldn't do anything. )

````
           <IndexedFaceSet  coordIndex="0 8 9 1 -1 1 9 10 2 -1 2 10 11 3 -1 3 11 12 4 -1 4 12 13 5 -1 5 13 14 6 -1 6 14 15 7 -1 7 15 8 0 -1">
           <Coordinate  point="-4.0 1.1 -2.0, -3.0 1.1 -3.0, 3.5 1.1 -3.0, 4.0 1.1 -2.5, 4.0 1.1 2.5, 3.5 1.1 3.0, -3.0 1.1 3.0, -4.0 1.1 2.0, -3.0 0.0 -1.6, -2.0 0.1 -3.0, 3.1 0.1 -3.0, 3.6 0.0 -2.0, 3.6 0.0 2.0, 3.1 0.1 3.0, -2.0 0.1 3.0, -3.0 0.0 1.6, "/>
           </IndexedFaceSet>
````

the files i have like this were all converted to x3d from vrml using "Vrml97ToX3dNist", which apparently always did this, (well in 2004 it did).  i think it used xslt, which i can see would be easier if you always have the comma after every Vec3f.


> Please consider to use a web server for Edge and Chrome, they do not serve the file:// protocol. Firefox should work fine in every case.

this was discovered when being served from github.io when i started using 4.2.5

**Comment by splace** [comment]
do you packport when this happens?

**Comment by create3000** [comment]
This fix will be released shortly with 4.2.6.

**Comment by splace** [comment]
now using 4.2.6 and is fixed
